Updated MRR ranks and linear transformation (#58)
thesis20/thesis_paper: sections/experiments/experiments-chapter.tex

\end{equation}
MRR \cite{MRR} is employed for the leave-one-out evaluation, defined in \Cref{eqn:mrr}.
For each user $u$ in $U$, MRR is calculated by finding the rank $r_u$ of the first relevant recommendation for each list, and then taking the mean of these ranks.
For example, if we recommend three items to a user, and the third recommendation is relevant, we get a rank of $1/3$ for this user.

> If the first recommendations for another user has a relevant item as the first item of the list, we assign a score of $1/1$.

Reviewer: ```suggestion
If the recommendation for another user contains a relevant item as the first item of the list, we assign a score of $1/1$.
```

> With this, we get a final MRR@3 score of $(1/3 + 1/1) / 2 = 0.67$.

Author: ```suggestion
With this, we get a final MRR@3 score of $(1/3 + 1/1) / 2 = 0.65$.
```

> With this, we get a final MRR@3 score of $(1/3 + 1/1) / 2 = 0.65$.

Author: ```suggestion
With this, we get a final MRR@3 score of $(1/3 + 1/1) / 2 = 0.66$.
```
